src/core_plugins/kibana/server/lib/import/import_dashboards.js

```diff
@@ -0,0 +1,33 @@
+import { flatten } from 'lodash';
+import { SavedObjectsClient } from '../../../../../server/saved_objects';
+
+const acceptableTypes = [
```

Comment:
What is this list derived from? I would personally prefer that it wasn't hard-coded if at all possible